pkg/util/policy.go

```diff
@@ -38,6 +38,20 @@ var ignoredResources = map[schema.GroupVersionKind]struct{}{
 	}: {},
 }
 
+func GetDefaultClusterPropagationPolicyTypeMeta() metav1.TypeMeta {
+	return metav1.TypeMeta{
+		APIVersion: "policy.karmada.io/v1alpha1",
+		Kind:       "ClusterPropagationPolicy",
+	}
+}
+
+func GetDefaultPropagationPolicyTypeMeta() metav1.TypeMeta {
+	return metav1.TypeMeta{
```

Comment:
prefer to remove to pkg/typemeta with var/const
```
PropagationPolicy = metav1.TypeMeta{
		APIVersion: "policy.karmada.io/v1alpha1",
		Kind:       "PropagationPolicy",
	}
```

usage:
```
typemeta.PropagationPolicy 
```

Reply:
May I ask, is using var/const instead of get for performance reasons?